Name: calendar-1minute-interval
Version: 1.3.1-01
Author: aric
Description: allows start and end times at 1 minute intervals, instead of 5 minute.

--- .orig/usr/palm/applications/com.palm.app.calendar/app/controllers/datetime-assistant.js
+++ /usr/palm/applications/com.palm.app.calendar/app/controllers/datetime-assistant.js
@@ -167,7 +167,7 @@
 	this.startTimeModel = {
 		time: new Date(this.targetModifiedTime.start)
 	};
-    this.controller.setupWidget('edit_startTime', {label: $L('Time'),modelProperty:'time'}, this.startTimeModel);
+    this.controller.setupWidget('edit_startTime', {label: $L('Time'),modelProperty:'time',minuteInterval:1}, this.startTimeModel);
     this.controller.listen('edit_startTime',Mojo.Event.propertyChange, this.startTimeCallback.bindAsEventListener(this));
 		
 	this.endTimeModel = {
@@ -173,7 +173,7 @@
 	this.endTimeModel = {
 		time: new Date(this.targetModifiedTime.end)
 	};
-    this.controller.setupWidget('edit_endTime', {label: $L('Time'), modelProperty:'time'}, this.endTimeModel);
+    this.controller.setupWidget('edit_endTime', {label: $L('Time'), modelProperty:'time',minuteInterval:1}, this.endTimeModel);
     this.controller.listen('edit_endTime',Mojo.Event.propertyChange, this.endTimeCallback.bindAsEventListener(this));
 	
 

